GoogleForCreators/web-stories-wp
# Editor: Enable improved video captions by default

## Context
Since work on VideoCaptions has been completed. it would be the correct time to enable it by default.

## Summary



## Relevant Technical Choices



## To-do



## User-facing changes



## Testing Instructions






This PR can be tested by following these steps:

1. Open the editor.
2. Add a new video to a story and add captions to it
3. Move it around and maybe add some other elements on top of it so it's only partially visible
4. Play the video in the editor
5. See that the captions are displayed on top of all other elements and always at the same position on the canvas
6. The captions appearance should be very close to what you see when previewing the story on the frontend


## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #9938
